packages/babel-plugin-transform-optional-catch-binding/src/index.js

```diff
@@ -0,0 +1,19 @@
+import syntaxOptionalCatchBinding from "babel-plugin-syntax-optional-catch-binding";
+
+export default function() {
+  return {
+    inherits: syntaxOptionalCatchBinding,
+
+    visitor: {
+      CatchClause(path) {
+        if (path.node.param.name) {
```

Comment:
I don't understand how this isn't throwing an error. If `path.node.param` is `null` (as babylon now sets), this is a nil access.